File: geolocator_platform_interface/lib/src/enums/location_accuracy_status.dart
/// Represent the current Location Accuracy Status on iOS 14.0 and higher.
enum LocationAccuracyStatus {
  /// A approximate location will be returned (Approximate location).
  reduced,

  /// The precise location of the device will be returned.
  precise,

  /// When an Android device is used, an 'unknown' status is returned, since
  /// Android does not support Approximate Location yet.
  unknown,
}

File: geolocator_android/android/src/main/java/com/baseflow/geolocator/location/LocationAccuracyStatus.java
package com.baseflow.geolocator.location;

public enum LocationAccuracyStatus {
  /// A approximate location will be returned (Approximate location).
  reduced,

  /// The precise location of the device will be returned.
  precise,

  /// When an 'unknown' status is returned, Location Permission is not yet requested or denied
  // (forever).
  unknown,
}
